### ISSUE TITLE
Keyboard support improvements in accessibility checker

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#10135]
Based on feedback from RNIB – the accessibility checker should support the following keyboard interactions:

- [ ] Press the Escape key to dismiss the checker UI
- [ ] Have a way to cycle back and forth between the panel with checker results, and the elements highlighted in the results
- [ ] (to double check) The dialog component should set `aria-expanded` on its toggle

The Escape key should have already been set up with usage of our `dialog` component but the behavior might be broken due to it being loaded within a web component.

The keys cycling might need further research – picking a custom keyboard shortcut to do this, implementing it correctly, and communicating what the shortcut is to users. Potentially also having a way to cycle without a keyboard shortcut.

**Comment by zerolab** [comment]
@thibaudcolas I believe https://alice.pages.igalia.com/blog/how-shadow-dom-and-accessibility-are-in-conflict/ could be relevant to this

**Comment by thibaudcolas** [comment]
This was implemented as part of #12187, where the errors are now rendered inline within the page for much better keboard support.
